proto/tendermint/store/types.proto

```diff
@@ -9,11 +9,14 @@ message BlockStoreState {
 }
 
 // TxInfo describes the location of a tx inside a committed block
-// as well as the result of executing the transaction.
+// as well as the result of executing the transaction and the raw log output.
 message TxInfo {
   int64 height    = 1;
   uint32 index    = 2;
   // The response code of executing the tx. 0 means
   // successfully executed, all others are error codes.
   uint32 code     = 3;
+  // The unprocessed log output generated during
+  // the execution of a transaction.
+  string raw_log   = 4;
```

Comment:
[question] is there a non-raw / processed form of the log? Other names in this PR call it `logs`, `response.Log` so if there is actually no difference between raw log and log, proposal to rename it to log for consistency:

```suggestion
  // The unprocessed log output generated during
  // the execution of a transaction.
  string log   = 4;
```